packages/node/src/handlers.ts

```diff
@@ -221,12 +222,21 @@ export function requestHandler(options?: {
   transaction?: boolean | TransactionTypes;
   user?: boolean | string[];
   version?: boolean;
+  wait?: boolean;
```

Comment:
Can we make this `number | undefined` instead of a `boolean` and rename it to `flushTimeout`.
Also, use `flushTimeout` then instead of the hardcoded `2000`.

Reply:
then maybe make it `number` and let `0` be the default which means "no flush"?

Reply:
this would also work

Reply:
I've updated the code